packages/frontend/app/(dashboard)/course/[cid]/queue/[qid]/components/QuestionCard.tsx

```diff
@@ -49,151 +56,206 @@ const QuestionCard: React.FC<QuestionCardProps> = ({
-            />
-          ))}
-        </Col>
-        {isBeingHelped && !isStaff && question.helpedAt && (
-          <Col flex="0 0 3rem">
-            <div className="text-sm font-medium text-green-700">
-              {servedTime}
-            </div>
+            {question.questionTypes?.map((questionType, index) => (
+              <QuestionTagElement
+                key={index}
+                tagName={questionType.name}
+                tagColor={questionType.color}
+              />
+            ))}
           </Col>
-        )}
-        <Col flex="0 0 3rem">
-          <div className="text-sm text-gray-600">{getWaitTime(question)}</div>
-          {isStaff && isBeingHelped && (
-            <div className="text-sm font-medium text-green-700">
-              {servedTime}
-            </div>
-          )}
-        </Col>
-        {isStaff && (
-          <Col className="w-full sm:w-auto">
-            <TAQuestionCardButtons
-              courseId={cid}
-              queueId={qid}
-              question={question}
-              hasUnresolvedRephraseAlert={false}
-              tasksSelectedForMarking={tasksSelectedForMarking}
-              className="align-center flex items-center justify-around"
-            />
+          <Col flex={'0.1 1 auto'}>
+            {(isBeingHelped || isPaused) && !isStaff && (
+              <Row justify={'end'}>
+                <div
+                  className={cn(
+                    'text-sm',
+                    isPaused ? 'text-amber-400' : '',
+                    isBeingHelped ? 'text-green-700' : '',
+                  )}
+                >
+                  {isPaused && 'Currently Paused'}
+                  {isBeingHelped && 'Currently Being Served'}
+                </div>
+              </Row>
+            )}
+            <Row
+              justify={'end'}
+              className={cn(
+                !isBeingHelped && !isPaused ? 'h-[2.5rem]' : '',
+                'gap-1',
+              )}
+            >
+              <Col flex="1 0 3rem">
+                {isStaff && (
+                  <div className="flex justify-end text-sm text-gray-600">
+                    {getWaitTime(question)}
+                  </div>
+                )}
+                {(isBeingHelped || isPaused) && (
+                  <div
+                    className={cn(
+                      isBeingHelped ? 'text-green-700' : '',
+                      isPaused ? 'text-amber-400' : '',
+                      'font-md flex justify-end text-sm',
+                    )}
+                  >
+                    {isPaused && (
+                      <>
+                        <div className={'text-gray-600'}>
+                          {getOriginalPausedTime(question)}
+                        </div>
+                        <div>{isPaused ? ' +' + pausedTime : ''}</div>
+                      </>
+                    )}
+                    {isBeingHelped && servedTime}
```

Comment:
I realised that resuming helping the question doesn't actually continue on from the old helptime, it just starts a new timer. 
![image](https://github.com/user-attachments/assets/ecac59ea-db77-4c37-8bc1-89b58383ff90)
![image](https://github.com/user-attachments/assets/e5f9c1ad-db10-4257-9858-5cb35933c307)
![image](https://github.com/user-attachments/assets/a5a8f8ff-22b6-4b7e-9f4a-3bb8c7b88c5e)

This was kinda why I linked #87 with this issue.

That's okay though, when I make the helpTime calculations more accurate, this will be solved.